Imagr/Utils.py

```diff
@@ -354,6 +355,29 @@ def downloadFile(url, additional_headers=None, username=None, password=None):
     return data, error
 
 
+def getDMGSize(url):
+    url_parse = urlparse.urlparse(url)
+    error = None
+    error = type("err", (object,), dict())
+    if url_parse.scheme in ['http', 'https']:
+        request = urllib2.Request(url)
+        try:
+            dmg = urllib2.urlopen(request)
+            data = dmg.headers['Content-Length']
+        except urllib2.URLError, err:
```

Comment:
These two except clauses could be combined:
```python
    except (urllib2.URLError, urllib2.HTTPError), err:
        setattr(error, 'reason', err)
        data = False
```

Reply:
fixed